testData/preferences/reset.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think we had decided that any new preference sets added to the system would be supplied with a text file explaining the purpose and scope of the file (that is, what contexts/platforms it is designed to be used in, and where it should end up once we organise our preference sets) - could you add a .txt or .md file alongside it explaining what was in your comment? Cheers